frontend/public/components/factory/list-page.jsx

```diff
@@ -199,7 +199,7 @@ export const FireMan_ = connect(null, { filterList })(
         if (createProps.to) {
           createLink = (
             <Link className="co-m-primary-action" {...createProps}>
-              <Button variant="primary" id="yaml-create">
+              <Button variant="primary" id="yaml-create" data-test-id="yaml-create">
```

Comment:
@dtaylor113 You said Cypress works better with `data-cy` attributes, right? We should start using those for new test attributes.

Reply:
Do we want to convert all our `data-test-id`'s to `data-cy`'s?  I wasn't sure when/if we wanted to do this, or if having some as `data-test-id` and some as `data-cy` would be confusing.

Reply:
> Do we want to convert all our `data-test-id`'s to `data-cy`'s? I wasn't sure when/if we wanted to do this, or if having some as `data-test-id` and some as `data-cy` would be confusing.

I wouldn't convert existing ones being used by Protractor tests (or at least not yet). But I think new ones only used by Cypress tests should be `data-cy`.

Reply:
fixed, started using `data-test`